src/Sentry.Extensions.AI/SentryAIUtil.cs

```diff
@@ -0,0 +1,22 @@
+using Sentry.Internal;
+
+namespace Sentry.Extensions.AI;
+
+internal static class SentryAIUtil
+{
+    internal static ISpan? GetActivitySpan()
```

Comment:
We could rename this as well, I think... to make it clearer what it does.
```suggestion
    internal static ISpan? GetFICCSpan()
```